ubuntu-7.4/rootfs/etc/nginx/cache/fastcgicache_skip_rules.conf

```diff
@@ -88,7 +88,13 @@ set_by_lua_block $cache_args {
         ngx.var.without = ngx.encode_args( without )
 
         -- Return accepted query var key-value pairs to be used in the cache key
-        return "?" .. table.concat( with, "&" )
+        local len = 0
+         
+        for k, v in pairs(with) do
+            len = len + 1
+        end
+        return len > 0 and "?" .. table.concat( with, "&" ) or ""
```

Comment:
Voi lua sentään kun tää pitää laskea käsin, eikä ole mitään sizea... :(

Reply:
Olisko tää kuitenkin nopeampi tehdä näin, vai meneekö sekavaksi?

```
        -- Return accepted query var key-value pairs to be used in the cache key

        for k, v in pairs(with) do
            return "?" .. table.concat( with, "&" ) or
        end

        return ""
 ```

Reply:
Kannattaako tämmöinen mikrosekuntien optimointi, kun samalla luettavuus kärsii aika paljon, että mitä tässä tehdään?